src/entity_extraction/spacy_entity_extraction.py

```diff
@@ -0,0 +1,52 @@
+# Author: Ty Andrews, Jenit Jain
+# Date: May 10, 2023
+
+import os
+import sys
+import spacy
+
+# ensure that the parent directory is on the path for relative imports
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
I think this was my mistake setting this up initially.

```suggestion
sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
```